integration_tests/cli/model-demo.sh

```diff
@@ -112,3 +112,132 @@ check_response "$result" "\"vid\": $vid"
+test_divider
+
+productLabel="Device #1"
+echo "Add Model with VID: $vid_in_hex_format PID: $pid_in_hex_format"
+result=$(echo "test1234" | dcld tx model add-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format --deviceTypeID=1 --productName=TestProduct --productLabel="$productLabel" --partNumber=1 --commissioningCustomFlow=0 --from=$vendor_account --yes)
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+test_divider
+
+echo "Get Model with VID: $vid_in_hex_format PID: $pid_in_hex_format"
+result=$(dcld query model get-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format)
+check_response "$result" "\"vid\": $vid"
+check_response "$result" "\"pid\": $pid"
+check_response "$result" "\"productLabel\": \"$productLabel\""
+echo "$result"
+
+test_divider
+
+echo "Get all models"
+result=$(dcld query model all-models)
+check_response "$result" "\"vid\": $vid"
+check_response "$result" "\"pid\": $pid"
+echo "$result"
+
+test_divider
+
+echo "Get Vendor Models with VID: ${vid_in_hex_format}"
+result=$(dcld query model vendor-models --vid=$vid_in_hex_format)
+check_response "$result" "\"pid\": $pid"
+echo "$result"
+
+test_divider
+
+echo "Update Model with VID: ${vid_in_hex_format} PID: ${pid_in_hex_format} with new description"
+description="New Device Description"
+result=$(echo "test1234" | dcld tx model update-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format --from $vendor_account --yes --productLabel "$description")
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+test_divider
+
+echo "Get Model with VID: ${vid_in_hex_format} PID: ${pid_in_hex_format}"
+result=$(dcld query model get-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format)
+check_response "$result" "\"vid\": $vid"
+check_response "$result" "\"pid\": $pid"
+check_response "$result" "\"productLabel\": \"$description\""
+echo "$result"
+
+test_divider
+
+echo "Update Model with VID: ${vid_in_hex_format} PID: ${pid_in_hex_format} modifying supportURL"
+supportURL="https://newsupporturl.test"
+result=$(echo "test1234" | dcld tx model update-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format --from $vendor_account --yes --supportURL "$supportURL")
+check_response "$result" "\"code\": 0"
+echo "$result"
+
+test_divider
+
+echo "Get Model with VID: ${vid_in_hex_format} PID: ${pid_in_hex_format}"
+result=$(dcld query model get-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format)
+check_response "$result" "\"vid\": $vid"
+check_response "$result" "\"pid\": $pid"
+check_response "$result" "\"supportUrl\": \"$supportURL\""
+echo "$result"
+
+test_divider
+
+echo "Delete Model with VID: ${vid_in_hex_format} PID: ${pid_in_hex_format}"
+result=$(dcld tx model delete-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format --from=$vendor_account --yes)
+echo "$result"
+
+test_divider
+
+echo "Query non existent model"
+result=$(dcld query model get-model --vid=$vid_in_hex_format --pid=$pid_in_hex_format)
+check_response "$result" "Not Found"
+echo "$result"
+
```

Comment:
Low test coverage!
Consider adding more tests to check all queries vid/pid

Reply:
Done! ✅